Gemfile

```diff
@@ -124,6 +125,7 @@ group :test do
   gem 'rspec-retry'
   gem 'rspec_junit_formatter'
   gem 'shoulda-matchers', '~> 4.0', require: false
+  gem 'tableparser', require: false
```

Comment:
In the specs, I wanted to be able to parse the `Terminal::Table`  output, and as far as I could tell, that gem doesn't have any parsing.

I happened to have my own smol gem (https://github.com/zachmargolis/tableparser) that does this, so I opted to use it here. Open to removing if anybody feels strongly!

It's a pretty limited gem, [I YARD doc'ed it as best I could](https://www.rubydoc.info/gems/tableparser/1.0.1/Tableparser#parse-class_method)

Reply:
I like the idea of using something one of us "owns" rather than introducing a 3rd-party dependency...  no objection here.